src/smt/process_assertions.cpp

```diff
@@ -316,7 +316,7 @@ bool ProcessAssertions::apply(AssertionPipeline& ap)
   Trace("smt") << " assertions     : " << ap.size() << endl;
 
   // ff
-  if (options().ff.ffDisjunctiveBit)
+  if (!options().ff.ffDisjunctiveBit)
```

Comment:
So this is slightly confusing. Why does this have to be false to be enabled? Can't we flip the option?

Reply:
Yeah, that sound reasonable. How about changing the name to `--ff-elim-disjunctive-bit` to make it clear that this option enables an elimination pass?

Reply:
Sounds good